src/save.jl

```diff
@@ -79,7 +79,8 @@ function save_results(results::Results, filename::String;
                 ),
             )),
         )
-    # For DC lines, storage power/energy, and load_shed, save only if nonempty
+    # For DC lines, storage power/energy, load_shed, and flexible demand,
+    # save only if nonempty
```

Comment:
I think indenting the second line of this comment would be clearer to show that it's all one sentence.

Reply:
Done.